apps/backend/src/libs/plugins/file-plugin/libs/types/file-plugin.d.ts

```diff
@@ -0,0 +1,8 @@
+import { type MultipartFile } from "@fastify/multipart";
+import "fastify";
+
+declare module "fastify" {
+	interface FastifyRequest {
+		uploadedFile?: MultipartFile;
```

Comment:
We already have .d.ts file, where we add User. Let's have one .d.ts file only (index.d.ts in backend folder)